rust/kernel/file.rs

```diff
@@ -39,12 +137,6 @@ impl File {
         unsafe { (*self.ptr).f_pos as u64 }
     }
 
-    /// Returns whether the file is in blocking mode.
-    pub fn is_blocking(&self) -> bool {
-        // SAFETY: `File::ptr` is guaranteed to be valid by the type invariants.
-        unsafe { (*self.ptr).f_flags & bindings::O_NONBLOCK == 0 }
```

Comment:
Thanks for noticing (and removing) this. It was a shortcut before we supported flags.